### PR TITLE
FEATURE: BB-1408 pivotTable subtotal styles

### PR DESCRIPTION
**Pull request by yanlipnican**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)(../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3358/ : FAILURE in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4156/ : SUCCESS in 15s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3359/ : FAILURE in 5m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4157/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3362/ : FAILURE in 1m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4160/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3363/ : FAILURE in 1m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4162/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3364/ : SUCCESS in 13m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4163/ : SUCCESS in 16s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3369/ : SUCCESS in 3m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4167/ : SUCCESS in 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3373/ : SUCCESS in 9m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4169/ : SUCCESS in 8s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3393/ : SUCCESS in 4m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4201/ : SUCCESS in 8s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1807/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1156/ : UNSTABLE in 19m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4202/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3397/ : SUCCESS in 3m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4206/ : SUCCESS in 10s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3398/ : SUCCESS in 4m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4207/ : SUCCESS in 6s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1808/artifact/test-storybook/html-report/index.html

**Comment by yanlipnican** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-933.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1158/ : SUCCESS in 33m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4209/ : SUCCESS in 13s

File: src/helpers/agGrid.ts [hunk 1]
```diff
@@ -554,8 +561,17 @@ export const executionToAGGridAdapter = (
...
+
     const rowData = minimalRowData.map((dataRow: Execution.DataValue[], dataRowIndex: number) =>
-        getRow(dataRow, dataRowIndex, columnFields, rowHeaders, headerItems[0], intl),
+        getRow(
+            dataRow,
+            dataRowIndex,
+            columnFields,
+            rowHeaders,
+            headerItems[0],
+            getSubtotalStyles(resultSpec.dimensions ? resultSpec.dimensions[0] : null),
```
Comment (strouhaljan) [inline src/helpers/agGrid.ts]: Shouldn't the fallback parameter value be `undefined`? `getSubtotalsStyles()` expects `AFM.IDimension` or nothing.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/927/ : FAILURE in 1m 55s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1812/ : SUCCESS in 27m 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3404/ : SUCCESS in 5m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/928/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/990/ : SUCCESS in 31m 17s

**Pull request merged**